tests/examples/global_param.image.STEHE.feb.txt

```diff
@@ -8,7 +8,7 @@ STARTDAY	1
 ENDYEAR 	1949
 ENDMONTH	1
 ENDDAY		10
-CALENDAR    PROLEPTIC_GREGORIAN
+CALENDAR    STANDARD
```

Comment:
Why made this change? I think the current Stehekin example forcing is in `PROLEPTIC_GREGORIAN` calendar (unless you've changed it recently)?